moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectEntity.js

```diff
@@ -106,7 +133,7 @@ const ProjectSummaryProjectSponsor = ({
         )}
         {!editMode && (
           <ProjectSummaryLabel
-            text={sponsor?.entity_name || "n/a"}
+            text={sponsor?.entity_name}
```

Comment:
I noticed a bug after this change. To replicate:

1. Go to a project that does not have a `Lead` set yet
2. Click the **Lead** field in the **Project Summary** form
3. Click the "X" icon on the far right of the field to cancel out
4. WSOD

This root of this bug is way beyond the scope of this issue because I noticed that in many instances we pass `text={<the value> || ""}` to `ProjectSummaryLabel` instead of handling nullish values in that component itself. For now, I think passing `text={sponsor?.entity_name || ""}` will fix this.

Reply:
thanks for catching this @mddilley! i just committed the fix you suggested. i think you're so right about reworking the `ProjectSummaryLabel` component to handle nullish values and removing this conditional from the parent components. i'll create a separate issue to do that.